plugins/outputs/azure_data_explorer/azure_data_explorer_test.go

```diff
@@ -84,17 +66,11 @@ func TestWrite(t *testing.T) {
 			},
 		},
 		{
-			name:         "Error in Mgmt",
-			inputMetric:  testutil.MockMetrics(),
-			createTables: true,
-			client: &fakeClient{
-				queries: make([]string, 0),
-				internalMgmt: func(f *fakeClient, ctx context.Context, db string, query kusto.Stmt, options ...kusto.MgmtOption) (*kusto.RowIterator, error) {
-					return nil, errors.New("Something went wrong")
-				},
-			},
-			createIngestor:  createFakeIngestor,
-			metricsGrouping: tablePerMetric,
+			name:            "SingleTable metric grouping type",
+			inputMetric:     testutil.MockMetrics(),
+			createTables:    true,
+			tableName:       "test1",
+			metricsGrouping: singleTable,
```

Comment:
The name of the test is different and I think `Error in Mgmt` just disappeared from the list... Is this intentional? If so, what is the reason?

Reply:
Hi @srebhan, 
Yes, the test is dropped. I have mentioned above that the `localClient/fakeClient` is not valid now and removed. Earlier Mgmt method was being tested via `localClient`. Now since this is removed, handling errors in Mgmt is beyond our scope.